src/pages/Account/MarketBreakdown/useExtractData.ts

```diff
@@ -0,0 +1,93 @@
+/** @jsxImportSource @emotion/react */
```

Comment:
This pragma isn't necessary because there are no components using the css prop in this file

Reply:
Indeed, good catch